index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Marketing Acessability website</title>
+  </head>
+
+  <!-- header section with nav bar containing links -->
+  <body>
+    <!-- nav - header tag-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- navigation bar with the website links -->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management">Online Reputation Management</a>
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </nav>
+  </header>
+    <!-- nav tag -->
+    <div class="hero"></div>
+    <div class="content">
+      <!-- section content in main tag -->
+      <div 
+        id="search-engine-optimization" class="search-engine-optimization">
+        <!-- add alt tag to describe images 6 in total -->
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left" alt="Search Engine Optimization"
+        />
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </div>
+      <div
+        id="online-reputation-management" class="online-reputation-management">
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          class="float-right" alt="Online Reputation Management"
+        />
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </div>
+      <div id="social-media-marketing" class="social-media-marketing">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          class="float-left" alt="Social Media Marketing"
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </div>
+    </div>
+      <div class="benefits">
```

Comment:
There's something wrong with your indentation here - have you got prettier set up on your VS code? Since the `div` on line 79 is a child of the `div` on 78, there should be an indentation at the start of the line. Prettier would fix this for you, as long as there is no problem with the rest of the code.

Reply:
I do have Prettier installed, added indentation to 78